apps/src/music/views/PatternAiPanel.tsx

```diff
@@ -273,9 +278,21 @@ const PatternAiPanel: React.FunctionComponent<PatternAiPanelProps> = ({
         )}
         {userCompletedTask === 'drawnDrums' && (
           <div className={styles.helpContainer}>
-            <div className={classNames(styles.help, styles.helpGenerate)}>
+            <div
+              className={classNames(
+                styles.help,
+                styles.helpGenerate,
+                generateState === 'error' && styles.helpGenerateError
+              )}
+            >
               Click this button and A.I. will generate more drums based on what
               you started.
+              {generateState === 'error' && (
+                <div className={styles.errorMessage}>
+                  <br />
+                  Something went wrong. Try again.
+                </div>
```

Comment:
Would it be possible to display details of the error here?  Mainly to help in situations where we can't open the developer console to see the actual error.  Or should we rely on server logs to find the error?

Reply:
We can look at server logs, but agreed it might be handy to see the error details here. Let me see if there's a good place to include this since it might be long (i.e. a stack trace)

Reply:
I'll go ahead and merge this so we can start collecting with metrics and will follow-up with an error detail display.